setup.py

```diff
@@ -50,7 +50,7 @@
 
 install_requires = [
     'future>=0.15.2',
-    'grpcio>=1.0rc1',
+    'grpcio>=1.0.0',
```

Comment:
Do we need to regenerate the auto-gen code with 1.0.0 to ensure everything still works correctly?